### PR TITLE
added table defs for arbitrum balancer v2

### PR DESCRIPTION
**Pull request by convexmagma**
## What?
added table defs for arbitrum balancer v2

## How? 
 I used [abi-parser](https://nansen-contract-parser-prod.web.app) to build the table definitions

### CODE REVIEW

File: parse/table_definitions_arbitrum/balancer/V2_Vault_event_ExternalBalanceTransfer.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "amount",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "Vault_event_ExternalBalanceTransfer"
```
Comment (medvedev1088) [inline parse/table_definitions_arbitrum/balancer/V2_Vault_event_ExternalBalanceTransfer.json:63]: Table name should match the file name

File: parse/table_definitions_arbitrum/balancer/V2_StablePool_event_AmpUpdateStopped.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        "dataset_name": "balancer",
+        "schema": [
+            {
+                "description": "",
+                "name": "currentValue",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "V2_MetaStablePool_event_AmpUpdateStopped"
```
Comment (medvedev1088) [inline parse/table_definitions_arbitrum/balancer/V2_StablePool_event_AmpUpdateStopped.json:30]: Should match the file name

Comment (convexmagma) [reply parse/table_definitions_arbitrum/balancer/V2_StablePool_event_AmpUpdateStopped.json]: @medvedev1088 Corrected and went through other files to correct same mistake.

File: parse/table_definitions_arbitrum/balancer/V2_Vault_event_PoolBalanceChanged.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "protocolFeeAmounts",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "Vault_event_PoolBalanceChanged"
```
Comment (kome12) [inline parse/table_definitions_arbitrum/balancer/V2_Vault_event_PoolBalanceChanged.json:74]: This one also needs to be updated to match the filename. Once this PR merges your next PR should automatically run the checks so you can catch this as soon as you submit a PR.

Comment (convexmagma) [reply parse/table_definitions_arbitrum/balancer/V2_Vault_event_PoolBalanceChanged.json]: @kome12 Just pushed the fix. If understand correctly, this current PR doesn't have the auto-check functionality enabled? I notice that workflows tend not to update once I push.

Comment (medvedev1088) [reply parse/table_definitions_arbitrum/balancer/V2_Vault_event_PoolBalanceChanged.json]: We've updated the repo settings. The workflows should run now.

**Comment by medvedev1088** [comment]
It's been deployed

**Comment by convexmagma** [comment]
@medvedev1088 @kome12 Thanks!

**Pull request merged**